SPECS/checkpolicy/checkpolicy.spec

```diff
@@ -0,0 +1,936 @@
+%define libselinuxver 2.9-1
+%define libsepolver 2.9-1
+
+Summary:       SELinux policy compiler
+Name:          checkpolicy
+Version:       2.9
+Release:       3%{?dist}
+License:       GPLv2
+Vendor:        Microsoft Corporation
+Distribution:  Mariner
```

Comment:
>Distribution [](start = 0, length = 12)

I've noticed this just now: this spec is missing the `URL` tag. First time I've seen that btw.:) I'm guessing `[https://github.com/SELinuxProject/selinux`](https://github.com/SELinuxProject/selinux`) is the right one.